app/services/state_file/state_information_service.rb

```diff
@@ -88,6 +90,7 @@ def state_intake_class_names
         tax_payment_info_url: "",
         tax_payment_url: "",
         tax_refund_url: "",
+        timezone: 'America/Phoenix',
```

Comment:
Sorry I didn't catch this before, but Arizona has a unique timezone because they generally don't observe daylight savings time (although the Navajo nation, which is partially in AZ, does observe it). Either way, it would be best to set the Idaho timezone using 'America/Boise'. (The other east coast states using New York is fine though)